go/vt/mysqlctl/s3backupstorage/s3.go

```diff
@@ -57,6 +58,11 @@ import (
 	"vitess.io/vitess/go/vt/servenv"
 )
 
+const (
+	sseCustomerPrefix = "sse_c:"
+	MaxPartSize       = 1024 * 1024 * 1024 * 5 // 5GiB - limited by AWS https://docs.aws.amazon.com/AmazonS3/latest/userguide/qfacts.html
```

Comment:
This is not used anywhere

Reply:
used below as mentioned in your other comment.